app/models/slide.rb

```diff
@@ -14,11 +15,19 @@ class Slide < ActiveRecord::Base
   belongs_to :user
   belongs_to :event
 
+  # validation macros
+  validates :title, :description, :user_id, :file, presence: true
+
+  # callbacks
+  # https://github.com/mperham/sidekiq/wiki/Problems-and-Troubleshooting#cannot-find-modelname-with-id12345
+  # after_commit :convert_file, :on => :create
+  after_create :convert_file
```

Comment:
I have search this problem for a while.
[this article](https://reinteractive.net/posts/238-sidekiq-is-too-fast) may help.